### PR TITLE
A task2 ... update editApplicant controller so applicant can be edited.

### PR DESCRIPTION
**Pull request by turnerror**

### CODE REVIEW

File: public/js/saveApplicant.js [hunk 1]
```diff
@@ -80,12 +86,15 @@ let makeApiRequest = async (data) => {
         body: JSON.stringify(data)
     })
         .then(response => {
+            console.log(response);
```
Comment (turnerror) [inline public/js/saveApplicant.js:89]: nice one christian

File: public/js/saveApplicant.js [hunk 2]
```diff
@@ -58,6 +58,11 @@ let errorMessage = (validationType) => {
 let getCompletedFormData = () => {
     const formData = document.querySelectorAll(".submitApplicant");
     let data = {};
+    const queryString = window.location.search;
+    const urlParams = new URLSearchParams(queryString);
+    const id = urlParams.get('id');
+    data['id'] = id;
```
Comment (bradleypurdue) [inline public/js/saveApplicant.js:64]: Does it not make more sense just to do `data['id'] = urlParams.get('id');`?

File: public/js/saveApplicant.js (hunk 1 above)
Comment (bradleypurdue) [reply public/js/saveApplicant.js]: nice one christian

File: public/js/saveApplicant.js [hunk 3]
```diff
@@ -69,8 +74,9 @@ let getCompletedFormData = () => {
     return data;
 };
 
-let makeApiRequest = async (data) => {
-    return fetch('./api/saveApplicant', {
+let makeApiRequest = async (data, type) => {
+    let path = './api/' + (type === '/addapplicant' ? 'saveApplicant' : 'editApplicant');
```
Comment (bradleypurdue) [inline public/js/saveApplicant.js:78]: `const`?

File: src/Models/ApplicantModel.php [hunk 4]
```diff
@@ -151,4 +151,47 @@ public function deleteApplicant($id)
...
+                            `phoneNumber` = :phoneNumber,
+                            `cohortId` = :cohortId,
+                            `whyDev` = :whyDev,
+                            `codeExperience` = :codeExperience,
+                            `hearAboutId` = :hearAboutId,
+                            `eligible` = :eligible,
+                            `eighteenPlus` = :eighteenPlus,
+                            `finance` = :finance,
+                            `notes` = :notes
+
```
Comment (SamCrudge) [inline src/Models/ApplicantModel.php:176]: white space?

File: public/js/saveApplicant.js (hunk 1 above)
Comment (SamCrudge) [reply public/js/saveApplicant.js]: nice one christian

**Pull request merged**